src/main/java/com/aetherteam/aether_genesis/block/GenesisBlocks.java

```diff
@@ -55,6 +56,9 @@ public class GenesisBlocks {
     public static final RegistryObject<WallBlock> SKYROOT_WOOD_WALL = register("skyroot_wood_wall", () -> new GenesisDoubleDropsWall(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.0F).sound(SoundType.WOOD)));
     public static final RegistryObject<WallBlock> STRIPPED_SKYROOT_WOOD_WALL = register("stripped_skyroot_wood_wall", () -> new WallBlock(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.0F).sound(SoundType.WOOD)));
 
+    public static final RegistryObject<Block> AETHER_CRAFTING_TABLE = register("skyroot_crafting_table", () -> new BlockAetherCraftingTable(Block.Properties.of(Material.WOOD, MaterialColor.WOOD).strength(2.5F).sound(SoundType.WOOD)));
+    public static final RegistryObject<Block> HOLYSTONE_FURNACE = register("holystone_furnace", () -> new BlockHolystoneFurnace(Block.Properties.of(Material.STONE, MaterialColor.STONE).strength(2.5F).requiresCorrectToolForDrops()));
```

Comment:
- Holystone Furnace strength should be 3.5F
- You probably don't need to include MaterialColor.WOOD and MaterialColor.STONE I think they are set up automatically from the Material.

Reply:
got it